requirements-dev.txt

```diff
@@ -21,3 +21,4 @@ aiounittest==1.4.2
 yamllint==1.33.0
 wlc==1.13
 coloredlogs==15.0.1
+python-dotenv==1.0.0
```

Comment:
note I needed to install dotenv to load the environment variables easily. I could parse it manually, but this seemed better.